code/js/rcalc-localization.js

```diff
@@ -0,0 +1,25 @@
+var set_locale_to = function(locale) {
+  if (locale) {
+    $.i18n().locale = locale;
+  }
+  $('title').i18n();
+  $('body').i18n();
+};
+
+jQuery(function() {
+  $.i18n().load({
+    'en': '../js/i18n/en.json',
```

Comment:
Please use it constant for it in some file path of file or something else.

Reply:
new file rcalc-constant-file-path.js created where I store const as EN_JSON_FILE_PATH